pretyping/inductiveops.ml

```diff
@@ -675,6 +675,9 @@ let arity_of_case_predicate env (ind,params) dep k =
 (* Inferring the sort of parameters of a polymorphic inductive type
    knowing the sort of the conclusion *)
 
+let univ_level_mem l s = match s with
+| Prop | Set | SProp -> false
```

Comment:
Isn't this wrong? eg on Set/Set it should return true

Reply:
Hmm, maybe. Is it really wrong though? I think it's just a fast-path to not generate an additional universe that will be unified immediately.